action/protocol/poll/protocol.go

```diff
@@ -187,10 +194,14 @@ func NewGovernanceChainCommitteeProtocol(
 	if err != nil {
 		log.L().Panic("Error when constructing the address of poll protocol", zap.Error(err))
 	}
-
+	ns, err := NewNativeStaking(cm, nativeStakingContract)
```

Comment:
This needs to be gated by another fork height?

Reply:
this is addressed in latest change